src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -32,6 +32,9 @@ CHANGELOG
  * Added a `InMemoryTransport` to Messenger. Use it with a DSN starting with `in-memory://`.
  * Added `framework.property_access.throw_exception_on_invalid_property_path` config option.
  * Added `cache:pool:list` command to list all available cache pools.
+ * Deprecated the `json_manifest_path` option in favor of `json.manifest_path`
+ * Added a new option to the version strategy called `json.manifest_allow_missing`
```

Comment:
Shouldn't this be `json_manifest.path` and `json_manifest.allow_missing` ?

Reply:
If we want to add more option to the json part we will need to change that part too, i'm more for keeping json.

Reply:
The version strategy is called `JsonManifest`, the key should reflect that, if we have to add more options to the json part they wouldn't have anything to do with this strategy and shouldn't be in the same key.

Reply:
understood, will change